docs/book/getting-started/core-concepts.md

```diff
@@ -86,17 +84,21 @@ In order to benefit from the aforementioned core concepts to their fullest exten
 
 First, in order to utilize _stack components_ that are running remotely on a cloud infrastructure, you need to deploy a **ZenML Server**, so that it can communicate with these stack components and run your pipelines.
 
-![Visualization of the relationship between code and infrastructure.](../.gitbook/assets/04_architecture.png)
+![Visualization of the relationship between code and infrastructure.](../.gitbook/assets/04\_architecture.png)
+
+#### Server Deployment
+
+In order to benefit from the advantages of using a deployed ZenML server, you can either choose to use the **ZenML Cloud** which provides a control plane for you to create managed instances of ZenML servers, or deploy it in your self-hosted environment.
 
 #### Metadata Tracking
 
 On top of the communication with the stack components, the **ZenML Server** also keeps track of all the bits of metadata around a pipeline run. With a ZenML server, you are able to access all of your previous experiments with the associated details. This is extremely helpful in troubleshooting.
 
 #### Secrets
 
-The **ZenML Server** also acts as a [centralized secrets store](../platform-guide/set-up-your-mlops-platform/use-the-secret-store/use-the-secret-store.md) that safely and securely stores sensitive data such as credentials used to access the services that are part of your stack. It can be configured to use a variety of different backends for this purpose, such as the AWS Secrets Manager, GCP Secret Manager, Azure Key Vault, and Hashicorp Vault.
+The **ZenML Server** also acts as a [centralized secrets store](../user-guide/advanced-guide/secret-management/secret-management.md) that safely and securely stores sensitive data such as credentials used to access the services that are part of your stack. It can be configured to use a variety of different backends for this purpose, such as the AWS Secrets Manager, GCP Secret Manager, Azure Key Vault, and Hashicorp Vault.
```

Comment:
I know you maybe didn't write this, but since we're here ;) maybe you could add URL links to the docs for all these individual components?

Reply:
We actually do not have separated links for these centralized secrets stores.